src/pages/StatusPage.vue

```diff
@@ -273,30 +274,29 @@
             {{ $t("deleteStatusPageMsg") }}
         </Confirm>
 
-        <component is="style" v-if="config.customCSS" type="text/css">
+        <component :is="style" v-if="config.customCSS" type="text/css">
```

Comment:
I think the original one is correct, `style` is a string.

After adding a `:`, it probably becomes a undefined variable `style`.

Recommend to ignore the eslint rule if they complain this.

Reply:
```
[vue/require-component-is]
Expected '<component>' elements to have 'v-bind:is' attribute
```

And we have 2 other places when we are using `:is`:
![image](https://user-images.githubusercontent.com/905878/166088129-b8754d3a-c620-4260-8f53-8400b07acc65.png)

Reply:
Just tested, this change actually bugged as I thought.

![image](https://user-images.githubusercontent.com/1336778/166131000-258f6ac4-2204-4c8b-b91a-a6d86b6740fd.png)

Reply:
Reverted. Thanks for testing